backend/src/main/java/com/example/backend/model/Sight.java

```diff
@@ -0,0 +1,26 @@
+package com.example.backend.model;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "sights")
+public class Sight {
+
+    @Id
+    private String id;
+
+    private String name;
+    private String image;
+    private String adresse;
+    private String webseite;
+    private String uhrzeit;
+    private String beschreibung;
+    private String lokation;
```

Comment:
Sieht soweit gut aus. Würde aber die Attribute auf englisch auflisten.
Das was man auf der App später im frontend als User lesen kann, kann ja auf deutscher Sprache sein.

```suggestion
    private String address;
    private String website;
    private String time;
    private String description;
    private String location;
```